### ISSUE TITLE
Progress bar

### ISSUE THREAD

**Issue by richardsweeney** [wp-cli/search-replace-command#134]
**Issue: I can't see the status of any search-replace commands as they are executing**

Right now I'm running a search-replace command (as a regular expression) against the postmeta table which has around 1 million rows. This is pretty slow (which obviously has nothing to do with this command in itself) but I'd love some kind of feedback as to the progress of the command.

For reference the command is:

`wp search-replace '\/vid-dodsfall\/(.)+' '/begravning/$1'  --regex wp_postmeta`

**Solution: I'd love to see a progress bar indicator for this command**

I would guess that the majority of use cases for `search-replace` are something along the lines of

`wp search-replace https://live.com https://local.com`

This can take a wee while on a larger database and I'd love to have some indication of progress.

**Comment by KratosGemini** [comment]
I agree, it would be nice. I searched Google if there was a way to do it and landed here.
